dirsrvtests/tests/suites/plugins/attruniq_test.py

```diff
@@ -80,4 +95,161 @@ def test_modrdn_attr_uniqueness(topology_st):
+    attruniq.enable_all_subtrees()
+    attruniq.enable()
+    topology_st.standalone.restart()
+
+    log.info('Create subtrees container')
+    containers = nsContainers(topology_st.standalone, DEFAULT_SUFFIX)
+    cont1 = containers.create(properties={'cn': EXCLUDED_CONTAINER_CN})
+    cont2 = containers.create(properties={'cn': EXCLUDED_BIS_CONTAINER_CN})
+    cont3 = containers.create(properties={'cn': ENFORCED_CONTAINER_CN})
+
+    log.info('Create test users')
+    users = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX,
+                         rdn='cn={}'.format(ENFORCED_CONTAINER_CN))
+    users_excluded = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX,
+                                  rdn='cn={}'.format(EXCLUDED_CONTAINER_CN))
+    users_excluded2 = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX,
+                                   rdn='cn={}'.format(EXCLUDED_BIS_CONTAINER_CN))
+
+    user1 = users.create(properties={'cn': USER_1_CN,
+                                     'uid': USER_1_CN,
+                                     'sn': USER_1_CN,
+                                     'uidNumber': '1',
+                                     'gidNumber': '11',
+                                     'homeDirectory': '/home/{}'.format(USER_1_CN)})
+    user2 = users.create(properties={'cn': USER_2_CN,
+                                     'uid': USER_2_CN,
+                                     'sn': USER_2_CN,
+                                     'uidNumber': '2',
+                                     'gidNumber': '22',
+                                     'homeDirectory': '/home/{}'.format(USER_2_CN)})
+    user3 = users_excluded.create(properties={'cn': USER_3_CN,
+                                              'uid': USER_3_CN,
+                                              'sn': USER_3_CN,
+                                              'uidNumber': '3',
+                                              'gidNumber': '33',
+                                              'homeDirectory': '/home/{}'.format(USER_3_CN)})
+    user4 = users_excluded2.create(properties={'cn': USER_4_CN,
+                                               'uid': USER_4_CN,
+                                               'sn': USER_4_CN,
+                                               'uidNumber': '4',
+                                               'gidNumber': '44',
+                                               'homeDirectory': '/home/{}'.format(USER_4_CN)})
+
+    UNIQUE_VALUE = '1234'
+
+    log.info('Create user with unique attribute')
+    user1.add('telephonenumber', UNIQUE_VALUE)
+    assert user1.present('telephonenumber', UNIQUE_VALUE)
+
+    log.info('Add exclude subtree')
+    topology_st.standalone.modify_s("cn=attruniq,cn=plugins,cn=config",
+                                    [(ldap.MOD_ADD, 'uniqueness-exclude-subtrees',
+                                      ensure_bytes(EXCLUDED_CONTAINER_DN))])
```

Comment:
I haven't found a method in attribute uniqueness plugin code for easy configuration of exclude subtrees similar to add_unique_subtree, would it be beneficial to create it in the scope of this PR? So far it would only be used in this test.

Reply:
I'm ok with you adding it in this PR, perhaps rework the title of the commit to focus on that change (so it's easier to track when looking at the git log)?

Reply:
Thank you, good idea! Amended.